### ISSUE TITLE
Make output fields readable by screen readers

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#3583]
[+1 from customer](https://3.basecamp.com/3600924/buckets/1966559/messages/676074626#__recording_1062478292)

**Comment by ebruchez** [comment]
With NVDA, on Windows, the following markup:

```xml
<output
    id="fr-view-component≡section-1-section≡grid-2-grid≡control-3-control≡≡c⊙1"
    class="xforms-output-output"
    aria-labelledby="fr-view-component≡section-1-section≡grid-2-grid≡xf-746"
    aria-describedby="fr-view-component≡section-1-section≡grid-2-grid≡xf-747"
    tabindex="0"
    role="textbox"
    aria-readonly="true">42</output>
```

produces the following results with:

- Microsoft Edge:
    - [LABEL] edit 42 readonly
- IE11
    - column 3 [LABEL] edit readonly
- Chrome
    - [LABEL] edit readonly 42
- Firefox
    - [LABEL] edit readonly 42

The only browser causing a real problem is IE11. It is unable to read the value at all as it doesn't know `<output>` at all. I thought that `aria-valuetext` might help, but no.

Remaining tasks:

- [ ] not all `<xf:output>` should be focusable
  - → make only the ones that have a label
- [ ] test that this works ok with the test forms

**Comment by ebruchez** [comment]
Mac OS with VoiceOver:

- Chrome
  - Non-repeated: 42 [LABEL] text
  - Repeated: 42 [LABEL] text
- Safari
  - doesn't stop on output field

**Comment by ebruchez** [comment]
Done, with limitations. See above.
